pkgs/top-level/python-packages.nix

```diff
@@ -1719,6 +1721,10 @@ in {
     inherit (pkgs.darwin.apple_sdk.frameworks) CFNetwork Security;
   };
 
+  ufonormalizer = callPackage ../development/python-modules/ufonormalizer { };
+
+  ufoProcessor = callPackage ../development/python-modules/ufoProcessor { };
```

Comment:
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)